spark/src/test/scala/org/apache/comet/exec/CometExecSuite.scala

```diff
@@ -63,23 +63,35 @@ class CometExecSuite extends CometTestBase {
     }
   }
 
-  test("Window range frame should fall back to Spark") {
+  test("Window range frame with Long") {
     val df =
       Seq((1L, "1"), (1L, "1"), (2147483650L, "1"), (3L, "2"), (2L, "1"), (2147483650L, "2"))
         .toDF("key", "value")
 
     checkAnswer(
       df.select(
         $"key",
-        count("key").over(
+        sum("key").over(
```

Comment:
So this test fails before this change?

Reply:
We currently don't support `count` so change to `sum`. Previously, even if the aggregate function is not supported, it still goes to the window frame code. In this PR, I return None directly if aggregate function is not supported.